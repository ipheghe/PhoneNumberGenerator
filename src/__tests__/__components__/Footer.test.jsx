// react libraries
import React from 'react';

// third-party libraries
import { shallow } from 'enzyme';
import render from 'react-test-renderer';

// components
import Footer from '../../components/Footer';

describe('Footer component', () => {
  it('should match component snapshot', () => {
    const tree = render.create(<Footer />);
    expect(tree).toMatchSnapshot();
  });

  it('renders without crashing', () => {
    const shallowWrapper = shallow(<Footer />);
    expect(shallowWrapper).toBeDefined();
    expect(shallowWrapper.find('.footer__container').length).toBe(1);
    expect(shallowWrapper.exists()).toBe(true);
  });
});
