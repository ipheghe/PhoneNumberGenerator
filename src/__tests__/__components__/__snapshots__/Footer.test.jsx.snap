// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match component snapshot 1`] = `
<div
  className="footer__container"
>
  <p>
    Copyright Â© 2019. All rights reserved.
  </p>
</div>
`;
